contracts/external/cover/CoverBroker.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity ^0.8.18;
+
+import "@openzeppelin/contracts-v4/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts-v4/security/ReentrancyGuard.sol";
+
+import "../../interfaces/ICover.sol";
+import "../../interfaces/ICoverBroker.sol";
+import "../../interfaces/IMemberRoles.sol";
+
+/// @dev Allows cover distribution by buying cover in behalf of the caller
+contract CoverBroker is ICoverBroker, ReentrancyGuard {
+
+  // Immutables
+  address owner;
```

Comment:
Use `Ownable` from OZ instead as it allows ownership change (for example if the wallet is compromised)

Reply:
added Ownable.sol inheritance